editor/components/block-toolbar/style.scss

```diff
@@ -19,40 +19,3 @@
 	display: inline-flex;
 	border-left: 1px solid $light-gray-500;
 }
```

Comment:
There's a mention of `.editor-block-toolbar__mobile-tools` in this file, I guess we can remove it right now?

Reply:
Fixed this one, and found it another place too, thanks.